.github/workflows/CI.yml

```diff
@@ -46,6 +46,17 @@ jobs:
     - name: Test - PAModel
       run: dotnet bin/Debug/PAModelTests/PAModelTests.dll
 
+    - name: Test - YamlValidator
+      run: dotnet bin/Debug/YamlValidator.Tests/YamlValidator.Tests.dll
+
+    - name: Test - Persistence.Tests Yaml Files (ubuntu & macos)
```

Comment:
note: you can simplify cross-platform scripting if you use PowerShell Core scripts. it makes it much simpler to implement and maintain when there's only one scripting language.

consider this for your next PR.